examples/rand-extension/lib.rs

```diff
@@ -130,5 +132,39 @@ mod rand_extension {
             let rand_extension = RandExtension::default();
             assert_eq!(rand_extension.get(), [0; 32]);
         }
+
+        #[ink::test]
+        fn chain_extension_works() {
+            // given
+            struct MockedExtension;
+            impl ink_env::test::ChainExtension for MockedExtension {
```

Comment:
Do I need to mock chain extension inside every test function? Is there anyway to define outside and use that in all functions?

Reply:
You can also define it outside, I just mocked it in here since it's only used in this one test.